test/singleline.js

```diff
@@ -92,14 +92,14 @@ var transforms = [
     msg: 'No Space before colon (key-spacing)'
   },
   {
-    str: 'var x = {key:123,more:456}\n',
-    expect: 'var x = {key: 123, more: 456}\n',
-    msg: 'Space after comma in keys'
-  },
-  {
     str: 'if(true){}\n',
     expect: 'if (true) {}\n',
     msg: 'Space after if'
+  },
+  {
+    str: 'var x = 123; // Useful comment\n',
+    expect: 'var x = 123 // Useful comment\n',
+    msg: 'Remove uneeded trailing semicolons that are followed by a comment'
```

Comment:
Haha, no worries. I submitted #63 which covers the typo and renames the test file.